lib/friendly_shipping/services/ups/serialize_address_snippet.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module FriendlyShipping
+  module Services
+    class Ups
+      class SerializeAddressSnippet
+        class <<self
+          def call(xml:, location:)
+            if location.company_name.present? # Is this a business address?
```

Comment:
Same ActiveSupport question applies here

Reply:
Removed from theses spots - `company_name` would be `nil` if unset, so `if location.company_name` will do the right thing here.